scripts/gen-s-parser.py

```diff
@@ -0,0 +1,408 @@
+#!/usr/bin/env python3.6
+#
+# Copyright 2018 WebAssembly Community Group participants
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+instructions = [
+    ("unreachable",    "allocator.alloc<Unreachable>()"),
```

Comment:
Please change this to `makeUnreachable` and add a tiny function doing that, I think it'll make the code as a whole more regular.